src/state/ducks/todos/todoSaga.js

```diff
@@ -1,26 +1,61 @@
-import { takeEvery, put } from "redux-saga/effects";
+import { takeEvery, put, call, takeLatest } from "redux-saga/effects";
+import { ApiCallTodos } from "../../../services/api.js";
 import {
+  fetchTodosRequest,
+  fetchTodosSuccess,
+  fetchTodosFail,
   addTodoRequest,
   addTodoSuccess,
+  addTodoFail,
   removeTodoRequest,
   removeTodoSuccess,
+  removeTodoFail,
   updateTodoRequest,
   updateTodoSuccess,
+  updateTodoFail,
 } from "./todoSlice.js";
 
+function* handleFetchTodos(action) {
+  try {
+    const data = yield call(ApiCallTodos, "", "GET", null);
```

Comment:
make it generic like this
```suggestion
    const data = yield call(apiCallRequest, "", "GET", null);
```